src/main.cpp

```diff
@@ -314,9 +314,11 @@ int main(int argc, char* argv[])
         // Check if we need to show our first run window.
         QSettings settings;
         int uiMode = settings.value(QStringLiteral("UiMode"), QJackTrip::UNSET).toInt();
+#ifndef __unix__
```

Comment:
is `__unix__` true for macOS?

Reply:
No. But it is true on BSD which is why we're using it here instead of just using __linux__.

Reply:
Thanks for the clarification!